httpserver/http_connection.cpp

```diff
@@ -592,6 +594,62 @@ void connection_t::handle_wrong_swarm(const std::string& pubKey) {
     BOOST_LOG_TRIVIAL(info) << "Client request for different swarm received";
 }
 
+constexpr auto DB_POLL_FREQUENCY = std::chrono::milliseconds(200);
+constexpr auto LONG_POLL_TIMEOUT = std::chrono::milliseconds(20000);
```

Comment:
I think it would be best practice to check the request headers if the clients is willing to use long-polling, not sure how exactly, but something like a keep-alive field or a max-timeout that is quite big?

Reply:
How about using a custom header, say `X-Loki-Long-Poll`?

Reply:
Sounds good

Reply:
done